packages/components/Notification/src/Notification.types.ts

```diff
@@ -3,12 +3,13 @@ import { IViewProps, ITextProps } from '@fluentui-react-native/adapters';
 import { ButtonProps } from '@fluentui-react-native/button';
 import { FontTokens, IBorderTokens, IColorTokens, LayoutTokens } from '@fluentui-react-native/tokens';
 import { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
+import { FlexStyle } from 'react-native';
 
 export const notification = 'Notification';
 export const NotificationVariants = ['primary', 'neutral', 'primaryBar', 'primaryOutlineBar', 'neutralBar', 'danger', 'warning'] as const;
 export type NotificationVariant = typeof NotificationVariants[number];
 
-export interface NotificationTokens extends LayoutTokens, IBorderTokens, IColorTokens, FontTokens {
+export interface NotificationTokens extends LayoutTokens, IBorderTokens, IColorTokens, FontTokens, FlexStyle {
```

Comment:
Why did you need to extend FlexStyle? I'm guessing `LayoutTokens` didn't have what you wanted?

Reply:
Yeah, LayoutTokens didn't include alignSelf